config/routes.yaml

```diff
@@ -6,6 +6,11 @@ game:
    methods: GET
    controller: App\Controller\GameController::show
 
+replace:
```

Comment:
Would it perhaps be clearer to name all the endpoints? `/game/replace`, `/game/tilt`, ...